modules/apps/transfer/types/keys.go

```diff
@@ -30,7 +30,7 @@ const (
 	// DenomPrefix is the prefix used for internal SDK coin representation.
 	DenomPrefix = "ibc"
 
-	// AllowAllPacketDataKeys holds the string key that allows all packet data keys in authz transfer messages
+	// AllowAllPacketDataKeys holds the string key that allows all memo strings in authz transfer messages
 	AllowAllPacketDataKeys = "*"
```

Comment:
Maybe this constant can also be renamed in v9? We could also add a new constant with a more accurate name for the back ports.

Reply:
I am going to rename it in a followup PR, just to make the backport easier.